src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -19,6 +50,87 @@ std::string NodeResources::DebugString() {
   return buffer.str();
 }
 
+std::string PrintVector(std::vector<double> &vector) {
```

Comment:
```suggestion
std::string DebugString(std::vector<double> &vector) {
```
Doesn't actually print, just returns a `std::string`

Reply:
Ok, I'll change it to DebugString

Reply:
When I read `PrintVector`, I expected it to write the string to stderr/stdout, but this just returns a string so I'd prefer to call it something like `DebugString`

Reply:
Ah ok. I changed it to `Vector2String`. Cannot call it `DebugString` as it is conflicting with the identical named method of other classes.